src/main/kotlin/com/terraformation/backend/accelerator/api/ProjectVotesController.kt

```diff
@@ -24,9 +29,9 @@ import org.springframework.web.bind.annotation.RestController
 @AcceleratorEndpoint
 @RequestMapping("/api/v1/accelerator/projects/{projectId}/votes")
 @RestController
-class ProjectVotesController() {
+class ProjectVotesController(private val voteStore: VoteStore) {
   @ApiResponse200
-  @ApiResponse403
+  @ApiResponse403("attempting to read votes without sufficient privilege")
```

Comment:
"Attempting to read votes without sufficient privilege." ? I see other descriptions in this format, capitalized with period.